pkg/backend/httpstate/client/client.go

```diff
@@ -492,7 +492,7 @@ func (pc *Client) BulkEncrypt(ctx context.Context, stack StackIdentifier,
 ) ([][]byte, error) {
 	req := apitype.BulkEncryptRequest{Plaintexts: plaintexts}
 	var resp apitype.BulkEncryptResponse
-	if err := pc.restCallWithOptions(ctx, "POST", getStackPath(stack, "bulk-encrypt"), nil, &req, &resp,
+	if err := pc.restCallWithOptions(ctx, "POST", getStackPath(stack, "batch-encrypt"), nil, &req, &resp,
```

Comment:
What an unfortunate discrepancy

Reply:
I mean we can rename them... if we're going to do it we better do it now.

Reply:
Yeah, I thought we were going with bulk on the service side but discussions landed on keeping in line with the route name.

Happy to do the mass rename to use bulk in all methods and types, but it will add some noise and will require fixing service-side usages too.

Reply:
Raise a PR to rename Bulk to Batch for the decrypt side to match, then update this PR likewise.